typescript-api/scripts/runtime-upgrade.ts

```diff
@@ -36,7 +36,7 @@ src/dancelight/interfaces/types-lookup.ts
 function hackXcmV5Support() {
     // For dancebox, replace "Lookup89" with "StagingXcmV5Junction"
     const danceboxFilePath = "src/dancebox/interfaces/types-lookup.ts";
-    hackTypeReplacement(danceboxFilePath, "Lookup89", "StagingXcmV5Junction", 8);
+    hackTypeReplacement(danceboxFilePath, "Lookup90", "StagingXcmV5Junction", 8);
```

Comment:
why is this needed?

Reply:
Otherwise typescript types generation returns an error

Reply:
Apparently "Lookup89" means this is the 89th type, but this PR adds a new type that happens to be before that, so now the xcm junction is the 90th type...